edxval/tests/test_api.py

```diff
@@ -1341,7 +1341,7 @@ def test_video_transcript_missing_attribute(self, mock_logger):
 
         mock_logger.warn.assert_called_with(
             "VAL: Required attributes are missing from xml, xml=[%s]",
-            transcript_xml
+            b'<transcript file_name="wow.srt" language_code="en" file_format="srt" provider="Cielo24"/>'
```

Comment:
Why we need bytestring here? Also why didn't we made the value of `transcript_xml` bytestring above?